### PR TITLE
[chore] Update `legend` to `demo`

### PR DESCRIPTION
**Pull request by motorina0**
some old `legend` text still present

### CODE REVIEW

File: docs/guide/installation.md [hunk 1]
```diff
@@ -513,7 +513,7 @@ To install using docker you first need to build the docker image as:
 ```
 git clone https://github.com/lnbits/lnbits.git
 cd lnbits
-docker build -t lnbits-legend .
+docker build -t lnbits-demo .
```
Comment (dni) [inline docs/guide/installation.md:516]: docker build -t lnbits/lnbits .

File: docs/guide/installation.md [hunk 2]
```diff
@@ -533,7 +533,7 @@ mkdir data
 Then the image can be run as:
 
 ```
-docker run --detach --publish 5000:5000 --name lnbits-legend -e "LNBITS_BACKEND_WALLET_CLASS='FakeWallet'" --volume ${PWD}/.env:/app/.env --volume ${PWD}/data/:/app/data lnbits-legend
+docker run --detach --publish 5000:5000 --name lnbits-demo -e "LNBITS_BACKEND_WALLET_CLASS='FakeWallet'" --volume ${PWD}/.env:/app/.env --volume ${PWD}/data/:/app/data lnbits-demo
```
Comment (dni) [inline docs/guide/installation.md:536]:
```suggestion
docker run --detach --publish 5000:5000 --name lnbits -e "LNBITS_BACKEND_WALLET_CLASS='FakeWallet'" --volume ${PWD}/.env:/app/.env --volume ${PWD}/data/:/app/data lnbits
```

File: lnbits/core/templates/core/index.html [hunk 3]
```diff
@@ -338,7 +338,7 @@ <h5 class="q-my-md">{{SITE_TAGLINE}}</h5>
           outline
           color="grey"
           type="a"
-          href="https://legend.lnbits.com/paywall/GAqKguK5S8f6w5VNjS9DfK"
+          href="https://demo.lnbits.com/paywall/GAqKguK5S8f6w5VNjS9DfK"
```
Comment (dni) [inline lnbits/core/templates/core/index.html:341]: i think this link is invalid now right? should be changed?

File: docs/guide/admin_ui.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ Account creation can be also disabled by setting `LNBITS_ALLOW_NEW_ACCOUNTS=fals
 
 ```
 $ sudo systemctl stop lnbits.service
-$ cd ~/lnbits-legend
+$ cd ~/lnbits-demo
```
Comment (dni) [inline docs/guide/admin_ui.md:47]:
```suggestion
$ cd ~/lnbits
```

File: lnbits/core/templates/core/index.html (hunk 3 above)
Comment (motorina0) [reply lnbits/core/templates/core/index.html]: done!

File: docs/guide/installation.md (hunk 1 above)
Comment (motorina0) [reply docs/guide/installation.md]: done

**Review by dni — APPROVED** [review]

**Pull request merged**